graphics/ipe-tools/Portfile

```diff
@@ -20,18 +19,20 @@ long_description        The following tools for the Ipe extensible drawing \
 
 homepage                http://ipe.otfried.org/
 
-checksums               sha1    a650856f7123fe68499f20bbe66695cb040306f7 \
-                        rmd160  9ee6d4eb84455d55a36d24e2233de9f6a25a21f7
+checksums           sha1    467cf36177a2fc6b6daf3410f43733bdff071c58 \
```

Comment:
The `sha1` checksum is no longer part of the recommended set - we can delete this line.